apps/backend/src/modules/auth/auth.controller.ts

```diff
@@ -35,6 +37,60 @@ class AuthController extends BaseController {
 				body: userSignUpValidationSchema,
 			},
 		});
+		this.addRoute({
+			handler: (options) =>
+				this.signIn(
+					options as APIHandlerOptions<{
+						body: UserSignInRequestDto;
+					}>,
+				),
+			method: "POST",
```

Comment:
Lets move methods to a separate enum so it can be reused then?

Reply:
I've renamed the { type HTTPMethod } to HTTPMethodType to avoid conflict with the newly added HttpMethod enum, which will be used more frequently. Let me know  if you have any other suggestions.

Reply:
That violates B17. I don't think we need this change. The method comes from HTTPMethod, meaning it is typed, so we can keep it as it was

Reply:
Should I roll back to { type HTTPMethod } and rename the enum, or change the type to something like HTTPRequestMethod while keeping the enum as HTTPMethod since it will be used more frequently?

Reply:
We don't need enum HTTPMethod. Roll back to have the same as we had before: `method: "POST",`